src/Sprite.cpp

```diff
@@ -35,6 +36,7 @@ Sprite::Sprite()
 	m_bSkipNextUpdate = true;
 	m_DecodeMovie = false;
 	m_EffectMode = EffectMode_Normal;
+	tab_tilde_scaling_enabled_ = false;
```

Comment:
I actually think we should just add this functionality on the parent actor class. I think many other actors can make use of this functionality and not just Sprites.

Reply:
Moved up to the Actor and I retested, all works as intended.